styles/components/position-hero.scss

```diff
@@ -0,0 +1,13 @@
+.position-hero {
+  position: relative;
+  width: 100%;
+  height: 40vh;
+  background: $c-position;
+
+  &__title {
+    position: absolute;
+    bottom: 50%;
+    left: 26%;
```

Comment:
Guessing % values to center something will never give consistent results. Instead stick to things like `50%` or `0` or `100%` for different centering techniques and then using transform translate with similar values (`50%` or `0` or `100%`) to center things.